templates/shared_services/sonatype-nexus-vm/terraform/locals.tf

```diff
@@ -1,8 +1,10 @@
 locals {
   core_vnet                = "vnet-${var.tre_id}"
   core_resource_group_name = "rg-${var.tre_id}"
-  nexus_allowed_fqdns      = "*pypi.org,files.pythonhosted.org,security.ubuntu.com,archive.ubuntu.com,keyserver.ubuntu.com,repo.anaconda.com,*.docker.com,*.docker.io,conda.anaconda.org,azure.archive.ubuntu.com,packages.microsoft.com,repo.almalinux.org,download-ib01.fedoraproject.org"
+  nexus_allowed_fqdns      = "*pypi.org,files.pythonhosted.org,security.ubuntu.com,archive.ubuntu.com,keyserver.ubuntu.com,repo.anaconda.com,*.docker.com,*.docker.io,conda.anaconda.org,azure.archive.ubuntu.com,packages.microsoft.com,repo.almalinux.org,download-ib01.fedoraproject.org,cran.r-project.org,cloud.r-project.org"
   nexus_allowed_fqdns_list = distinct(compact(split(",", replace(local.nexus_allowed_fqdns, " ", ""))))
+  workspace_vm_allowed_fqdns = "r3.o.lencr.org,x1.c.lencr.org,*.digicert.com,ocsp.godaddy.com,crl.godaddy.com"
```

Comment:
Are these fqdns going to be consistent for all customers? We could do with a list of common CRLs.

Reply:
I have found this in Azure docs [Azure TLS certificate changes](https://learn.microsoft.com/en-us/azure/security/fundamentals/tls-certificate-changes#will-this-change-affect-me)

Would you like me to add all of the listed URLs to workspace_vm_allowed_fqdns?
http://crl3.digicert.com
http://crl4.digicert.com
http://ocsp.digicert.com
http://www.d-trust.net
http://root-c3-ca2-2009.ocsp.d-trust.net
http://crl.microsoft.com
http://oneocsp.microsoft.com
http://ocsp.msocsp.com
http://www.microsoft.com/pkiops

Reply:
@JaimieWi I think @marrobi is wondering where we got the godaddy domains, and whether these domains will work for all users of the TRE (i.e. they're recognised FQDNs required for CRAN to work)?

Reply:
@marrobi @martinpeck I spent some time testing which CRLs are required for the R installation and it looks like it is actually only the two Lets Encrypt FQDNs. I have now edited the locals.tf file to have only these two

Reply:
Yes it seems so, it may need to be configurable for others depending on requirements. Is there anything you'd like me to change in this PR?

Reply:
@marrobi - I agree that allowing the CRL allowed domains in makes sense. My inclination would be to get this PR merged, and track making it configurable in an issue for a future PR to address?

Reply:
Tracking in #2746